### DISCUSSION TITLE
Correspondent Input with Autocomplete instead of long list

### DISCUSSION THREAD

**Discussion by thoughtgap** [LeoKlaus/Paperparrot#189] [Feature Requests]
My long (426) list of correspondents is not very user friendly. Please add text field with autocomplete, like paperless' web version. [Here's a demo with Swift.](https://github.com/simonloewe/TextFieldInputPrediction).

**Comment by LeoKlaus** [comment]
You can check out the new menus in the TestFlight beta.

**Reply by thoughtgap** [reply]
Supercool, thank you! Looks a bit tiny/fidgety at first, but works well. Thanks!

**Reply by LeoKlaus** [reply]
IMO it's not great on touchscreens, especially if a document has a bunch of tags. I'm still working on a solution to show tags in multiple rows when necessary.
